Doc/whatsnew/3.11.rst

```diff
@@ -281,11 +282,8 @@ time
 * On Windows, :func:`time.sleep` now uses a waitable timer which has a
   resolution of 100 nanoseconds (10\ :sup:`-7` seconds). Previously, it had
   a resolution of 1 millisecond (10\ :sup:`-3` seconds).
```

Comment:
Hum, you should mention "high-resolution timers" somewhere, since it's not supported by all Windows versions.

Moreover, "high-resolution timers" should also be mentioned in the time.sleep() documentation which now contains details about the imlpementation:
https://docs.python.org/dev/library/time.html#time.sleep